File: CucumberJVM/Chapter13/13.1-TheLocalScenarioContextPattern/Before/src/main/java/com/wimp/localscenariocontext/app/models/Order.java
package com.wimp.localscenariocontext.app.models;

public class Order {
    private final int orderNo;
    private final String customerName;
    private final String pizzaName;
    private OrderStatus status = OrderStatus.NEW;

    public Order(int orderNo, String customerName, String pizzaName) {
        this.orderNo = orderNo;
        this.customerName = customerName;
        this.pizzaName = pizzaName;
    }

    public int getOrderNo() {
        return orderNo;
    }

    public String getCustomerName() {
        return customerName;
    }

    public String getPizzaName() {
        return pizzaName;
    }

    public OrderStatus getStatus() {
        return status;
    }

    public void setStatus(OrderStatus status) {
        this.status = status;
    }
}

File: Reqnroll/Chapter13/13.1-TheLocalScenarioContextPattern/Before/WIMP.LocalScenarioContextSample.App/Models/Order.cs
namespace WIMP.LocalScenarioContextSample.App.Models;

public class Order(int orderNo, string customerName, string pizzaName)
{
    public int OrderNo { get; } = orderNo;
    public string CustomerName { get; } = customerName;
    public string PizzaName { get; } = pizzaName;
    public OrderStatus Status { get; set; } = OrderStatus.New;
}
